src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -515,4 +515,4 @@ doclet.NoFrames_specified=\
 # L10N: do not localize the option name -footer
 doclet.footer_specified=\
     The -footer option is no longer supported and will be ignored.\n\
-    It may be removed in a future release.
\ No newline at end of file
+    It may be removed in a future release.
```

Comment:
I believe this is to fix no newline at the end (unrelated to l10n changes). Do we need to change the copyright year for this?

Reply:
Actually I was correcting the L217, changed {) -> {}. But 2 days ago, Jonathan Gibbons fixed it in another commit 6bb6093. I found his fix after running the merge. Looks both of us forgot to update the copyright year. Any suggestion? 

doclet.help.systemProperties.body=\
    The {0} page lists references to system properties.

Reply:
I believe your PR still contains the fix to add the newline at the end of the file (at L518). So I think you can simply change the copyright year in `standard.properties` file.

Reply:
Thx! Fixed the copyright year for this file.